test/parallel/test-crypto-verify-failure.js

```diff
@@ -19,36 +19,32 @@ const options = {
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-const server = tls.Server(options, function(socket) {
-  setImmediate(function() {
-    console.log('sending');
+const server = tls.Server(options, (socket) => {
+  setImmediate(() => {
     verify();
-    setImmediate(function() {
+    setImmediate(() => {
       socket.destroy();
     });
   });
 });
 
 function verify() {
-  console.log('verify');
   crypto.createVerify('RSA-SHA1')
     .update('Test')
     .verify(certPem, 'asdfasdfas', 'base64');
 }
 
-server.listen(0, function() {
+server.listen(0, () => {
```

Comment:
I guess technically, this should be wrapped in a `common.mustCall()` to make sure the inner `common.mustCall()`s are executed.